es6-classes/square.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable no-undef -- Shape is imported by .html */
+/* exported Square */
+
+class Square extends Shape {
+  constructor(area, circumference, width) {
```

Comment:
Per instruction:
Write its constructor to take the **square's width as a parameter** (and nothing else)